src/css/style.css

```diff
@@ -0,0 +1,47 @@
+body {
+
+	background-color: darkgrey;
+	text-align: center;
+
+}
+
+#mainDiv {
+
+	margin-top: 200px;
+
+}
+
+h1 {
+
+}
+
+#textDiv {
+
+}
+
+form {
+
+}
+
+#formId {
+
+}
+
+#textInput {
+
+}
+
+#buttonId {
+
+}
+
+.stop {
+
+	display: none;
+
+}
+
+#image{
+	width: 500px;
+	height: 300px;
+}
```

Comment:
Like Scott mentioned, you'll want to get rid of the unused styles, plus indent the properties. And lint that sucker!